actionpack/test/dispatch/static_test.rb

```diff
@@ -160,6 +160,9 @@ def test_serves_gzip_files_when_header_set
     response  = get(file_name, 'HTTP_ACCEPT_ENCODING' => 'GZIP')
     assert_gzip  file_name, response
 
+    response  = get(file_name, 'HTTP_ACCEPT_ENCODING' => 'compress;q=0.5, gzip;q=1.0')
```

Comment:
We didn't have any tests where `gzip` wasn't the first content-coding, but now we're using the parsed accept encoding (rather than the raw string) it feels valuable to check.